tensorflow/c/c_api_internal.h

```diff
@@ -179,6 +180,12 @@ struct TF_ApiDefMap {
   tensorflow::mutex lock;
 };
 
+struct TF_Server {
```

Comment:
Ownership of the `server` pointer is probably best expressed using `std::unique_ptr`. So something like:

```c++
struct TF_Server {
  std::unique_ptr<tensorflow::ServerInterface> server;
}
```